static/sass/styles.scss

```diff
@@ -238,3 +238,13 @@ pre {
 .u-margin-large--top {
   margin-top: $sp-large !important;
 }
+
+.p-navigation {
+
+  &__logo {
+    @media (min-width: $breakpoint-navigation-threshold + 1) {
+      margin-top: .75rem !important;
+      margin-bottom: .25rem !important;
```

Comment:
We shouldn't need this. Can you file an issue on vanilla and add a comment here please.

Reply:
Issue created: https://github.com/vanilla-framework/vanilla-framework/issues/1782

Reply:
Sorry, I took that too literally :)
Done